scripts/deploy.py

```diff
@@ -18,6 +18,20 @@
 
 
 if __name__ == '__main__':
+
+    print("Decrypting secrets")
+
+    subprocess.check_call(
+        "openssl aes-256-cbc -K $encrypted_39cb4cc39a80_key "
+        "-iv $encrypted_39cb4cc39a80_iv -in secrets.tar.enc "
+        "-out secrets.tar -d",
+        shell=True
```

Comment:
Yeah, I don’t fancy unpicking the Travis logic any further, just registering a disquiet.